### ISSUE TITLE
Main activity: Show pictures to upload, rather than already uploaded pictures

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#371]
When the user opens the app, how about showing the user's recent pictures?

Benefits:
- Incitate people to upload
- Use less network data

The existing Gallery button would still be used when the user wants to upload a picture which is not recent.
Not sure how to get recent pictures, but a lot of apps do that. We could filter out those too small, those with no EXIF, and those already uploaded.

What do you think about it?

**Comment by nextLane** [comment]
Hey, need some clarity here. Recent pictures uploaded by the user himself/herself or the recent pictures uploaded from his/her region?
Please correct me if I am wrong, but presently we do show the recent uploads on top of the list right, do we intend to cut short on that list?

**Comment by nicolas-raoul** [comment]
You are right, the current main activity shows recently uploaded pictures.

For some time we have been considering replacing this main activity with a map of nearby needed pictures, but the idea I described above is probably also worth considering.

Nothing is decided yet, just discussing, so we need everyone's opinion :-)

**Comment by misaochan** [comment]
It sounds good to me, but if we sufficiently improve the Nearby function, I would personally prefer using that as the main activity to this. :) But showing recently-taken pictures could have its merits (not sure how we would do that though).

There is also the possibility that some people just enjoy seeing their recently-uploaded photos on the main screen, haha. I guess when the time comes to make a decision, we could set up a poll or something and get user input on their preferences?

**Comment by nicolas-raoul** [comment]
Showing nearby first is a good idea too to encourage uploads.

If there is much opposition, maybe we could open the last activity that people were using?

**Comment by chrisdebian** [comment]
Hi, all.

I think the 'nearby' functionality currently exists. Is this still an issue?

Thanks,

Chris.

**Comment by nicolas-raoul** [comment]
@chrisdebian I added a clarification.
